src/data/team.ts

```diff
@@ -73,6 +74,16 @@ export const getMembers = memoize(async () => {
           member.manager = omit(manager, "manager");
         }
       }
+      const reports = uniqBy(
+        parsed
+          .filter((m) => m.reports_to === member.name)
+          .map((report) => omit(report, ["manager", "reports"])),
+        "name"
+      );
```

Comment:
I updated this logic to ensure reports are duplicated. The sheet still has double entries for some people and they'd show up twice here.